jenkins-design-language/src/js/components/status/SvgSpinner.jsx

```diff
@@ -6,6 +6,33 @@ import { describeArcAsPath } from '../SVG';
 export const strokeWidth = 3.5; // px. Maybe we can fetch this from CSS at runtime in the future
 
 export default class SvgSpinner extends Component {
+    componentWillMount() {
+        this.infiniteRotationRunning = false;
+        this.setState({
+            infiniteRotatePercentage: 0
+        });
+    }
+
+    infiniteLoadingTimer = () => {
+        let infiniteRotatePercentage = this.state.infiniteRotatePercentage;
+        
+        infiniteRotatePercentage += 1.5;
+
+        if (infiniteRotatePercentage >= 360) {
+            infiniteRotatePercentage = 0;
+        }
+
+        this.setState({
+            infiniteRotatePercentage: infiniteRotatePercentage
+        });
+
+        requestAnimationFrame(this.infiniteLoadingTimer);
+    }
+
+    componentWillUnmount() {
+        cancelAnimationFrame(this.infiniteLoadingTimer);
```

Comment:
This needs the integer handle returned from `requestAnimationFrame()` like `setTimeout()`, rather than the handler.